spec/services/question_saver_spec.rb

```diff
@@ -39,6 +39,28 @@
     end
 
     context 'when there is a room on the question' do
+      it 'persists form data to the selected answer store' do
```

Comment:
Is this test and the one below the wrong way round? (i.e. this one is in the `room` context, but tests that the `problem` is persisted?)

Reply:
Ack, yes, I copied the block and then changed the wrong one. Thanks for catching that